e2e/components/Checklist/Checklist-test.avt.e2e.js

```diff
@@ -20,5 +20,38 @@ test.describe('Checklist @avt', () => {
       },
     });
     await expect(page).toHaveNoACViolations('Checklist @avt-default-state');
+
+    const buttonEle = page.locator('[aria-label="Checklist toggle"]');
+    const viewButton = page.getByRole('button', { name: 'View all (10)' });
+    //press tab to move focus to buttonELement
+    await page.keyboard.press('Tab');
+    await expect(buttonEle).toBeInViewport();
+    await expect(buttonEle).toBeFocused();
+
+    //check the checklist is expanded
+    await expect(buttonEle).toHaveAttribute('aria-expanded', 'true');
+    await expect(viewButton).toBeVisible();
+    //Press enter to collapse the checklist
+    await page.keyboard.press('Enter');
+    await expect(buttonEle).toHaveAttribute('aria-expanded', 'false');
+
+    //Press escape to clear the focus
+    await page.keyboard.press('Escape');
```

Comment:
The current behavior observed in the checklist is that pressing the "Escape" key only dismisses the tooltip, while the focus remains on the button. Since this is not a modal element, it is not intended to clear the focus when "Escape" is pressed.

Reply:
yes, its for dismissing the tooltip I forgot to update comment.